content/md/what-is/what-about-reference-entities.md

```diff
@@ -0,0 +1,39 @@
+---
+id: what-about-reference-entities
+themes: first-steps
+title: What about **reference entities**?
+popular: true
+ee-only: true
+related:
+---
+
+# What's the use case?
+
+Let's take an example of a product page.
+There are many types of product information such as text, images...
+Some information is shared between different products (such as the definition of care instructions, colors or brands).
+This shared information has its own attributes (eg a label, a description, an image for the brand).
+They may have dedicated pages on the e-commerce website (eg dedicated pages for brands) or their information may enrich each product page.
+
+# What's a reference entity?
+
+A **reference entity** allows you to create and enrich natively in the PIM those **common information** which are **linked to products**. 
+A reference entity has its **own attributes**, its **own lifecycle** and can be linked to products.
+It allows enriching common data related to products with a rich content (description, images...), more complex than just a code and labels.
+
+For example, you can create a reference entity to manage brands, ranges, manufacturers, colors, materials, care instructions...
+
+Let's take an example with an entity "Brand" and a list of brands (Kartell, Alessi, Fatboy, Fermob...). A brand is described by the following information: a code, a label, an image, a country, a designer, a foundation date, a photo and a description.
+
+
+# What's a record?
+
+A record contains all information about a reference entity. For example for the "Brands" reference entity, a record could be the information regarding the brand "Kartell".
+A record may be related to one or more products.
+
+# How to create your 1st reference entity and its records?
+
+From a single place, in a dedicated "Entities" menu, you can create your first reference entity, define its records structure, create and enrich its records. Let's go!
+1.  [Create your first family variant](/articles/manage-your-families.html#create-a-family-variant)
```

Comment:
We agree those links have nothing to do with ref entities, right?

Reply:
Yes, I added the links, but broken for the moment